repos/keg-components/src/components/form/checkbox/checkbox.js

```diff
@@ -45,45 +45,61 @@ const Input = StyleInjector(KegInput, {
   className: 'keg-checkbox'
 })
 
-const Element = React.forwardRef(
-  (
-    { className, elProps, styles = noPropObj, icon, checked, ...props },
-    ref
-  ) => {
-    const checkStyle = useMemo(() => {
-      return {
-        ...styles.indicator,
-        ...checkBoxStyles.icon,
-      }
-    }, [ checkBoxStyles, styles ])
+const Element = React.forwardRef((props, ref) => {
+  const {
+    className,
+    elProps,
+    styles = noPropObj,
+    CheckIcon = Check,
+    checked,
+    ...remainingProps
```

Comment:
I like calling these `attrs` for `attributes`. Cause really that's what they are

Reply:
updated